### DISCUSSION TITLE
Updates

### DISCUSSION THREAD

**Discussion by teamtacho** [liedekef/events-made-easy#264] [General]
Hi Franky

I have done the update to 2.4.2 as you described and it worked perfectly.
Now I see in GitHub that there is a new version 2.4.5. In wordpress/plugins there is no hint that a new version is available.
1. How will I get informed about new versions?
2. How shall I do the update? 
a. again with the new zip file like I have done for the "big upgrade"
b. via FTP

**Comment by liedekef** [comment]
Go to wordpress updates in your menu, it will appear there. There is a scheduled check for available updates planned twice a day, so normally after at most 12 hours the update will also appear automatically. It is a small limitation due to the update library I use, but should not pose any problems. I manage 11 sites, updates are managed perfectly "as usual" the WP way.

**Comment by teamtacho** [comment]
Under WP/updates no events made easy update is displayed. If I click on look for updates (link) the following messages appears:
Could not determine if updates are available for Events Made Easy.

GitHub API error. Base URL: "/repos/:user/:repo/releases/latest", HTTP status code: 403. puc-github-http-error
GitHub API error. Base URL: "/repos/:user/:repo/tags", HTTP status code: 403. puc-github-http-error
GitHub API error. Base URL: "/repos/:user/:repo/branches/master", HTTP status code: 403. puc-github-http-error
Could not retrieve version information from the repository. This usually means that the update checker either can't connect to the repository or it's configured incorrectly. puc-no-update-source

**Reply by liedekef** [reply]
That's weird. If you look at github release statistics, people can download/update just fine: https://tooomm.github.io/github-release-stats/?username=liedekef&repository=events-made-easy
Now: can it be some firewall/security settings at your end are blocking access to github?
